tests/converter/test_df_converter.py

```diff
@@ -85,5 +85,14 @@ def test_df_converter():
     conv.assert_intent("accountspendingcheck_en")
     conv.assert_frame({})
 
+    conv.process("transfer money")
+    conv.assert_text("Sure. Transfer from which account?")
+
+    conv.process("checking account")
+    conv.assert_text("To which account?")
+
+    conv.process("transfer to savings account")
```

Comment:
nice! does the follow up logic work from here when you specify the amount that you want to transfer?

Reply:
No, since there is not enough NLP data for amount role classification. We will not fix the NLP issue in this PR.

Reply:
Fixed this issue, looked like it was a bug in the annotation generator.